### PR TITLE
 added the html_graphics Tag

### PR DESCRIPTION
**Pull request by theanimator20**
Hello @Diptenusarkar ,

i have done the work assigned to me please consider merging the PR

Thank You !!
hop

### CODE REVIEW

**Comment by theanimator20** [comment]
@Diptenusarkar  Once verified do accept my PR with an hacktoberfest label
Happy Coding !!

**Review by Diptenusarkar — APPROVED** [review]

**Comment by Diptenusarkar** [comment]
Remove Conflicting

**Comment by theanimator20** [comment]
> Remove Conflicting

✅

**Comment by theanimator20** [comment]
@Diptenusarkar  shall i close the pull request?? Or u will do it

File: html/html_forms/index.html [hunk 1]
```diff
@@ -163,6 +163,9 @@ <h2>Button</h2>
         </div>
 
         <div class="navigation-link">
+
+            <a href="../html_colors/"><i class="bi bi-chevron-double-left"></i> Previous</a>
```
Comment (Diptenusarkar) [inline html/html_forms/index.html:167]: Why add extra button

**Review by Diptenusarkar — CHANGES_REQUESTED** [review]
Fix all problem 
Screenshot (1213)
Screenshot (1212)
Screenshot (1211)

**Comment by theanimator20** [comment]
Okk

**Comment by theanimator20** [comment]
@Diptenusarkar  solved the requested changes

**Comment by theanimator20** [comment]
Done  ✅ ✅

**Comment by Diptenusarkar** [comment]
Please Check link Page.

**Comment by theanimator20** [comment]
> Please Check link Page.

What's the issue with link page it's working fine right?

**Comment by Diptenusarkar** [comment]
Screenshot (1214)

**Comment by theanimator20** [comment]
@Diptenusarkar  i have resolved the issues kindly go through it

**Comment by Diptenusarkar** [comment]
Good Job 👍

**Pull request merged**